src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Windows.txt

```diff
@@ -0,0 +1,55 @@
+﻿@ECHO OFF
+SETLOCAL
+
+SET PACKAGE_DIR=%1
+SET EXECUTION_DIR=%2
+
+IF DEFINED PACKAGE_DIR ( echo Using %PACKAGE_DIR% as folder for resolving package dependencies.) ELSE (
+echo Please specify an package source directory using PACKAGEROOT parameter
+goto ShowUsage
+)
+if "%EXECUTION_DIR%" == "" (set EXECUTION_DIR=%~dp0)
+echo Executing in %EXECUTION_DIR% 
+
+:: ========================= BEGIN Copying files  =============================== 
+IF NOT EXIST %EXECUTION_DIR% ( md %EXECUTION_DIR%)
+
+echo Hard linking dependent files... 
+:: Format here is: call :copyandcheck Path1 Path2 || GOTO EOF
+[[CopyFilesCommands]]
+
+:: Confusing slash on CD here is intentional:
+:: With the switch to ~dp0, one gets a \ and one does not.
+IF /I "%CD%\" NEQ "%EXECUTION_DIR%" (
+xcopy /e /q /y %CD%\*.* %EXECUTION_DIR%
+)
+echo Finished linking needed files, moving to running tests.
+:: ========================= END Copying files  =================================
+
+:: ========================= BEGIN Test Execution ============================= 
+echo Running tests... Start time: %TIME%
+echo Command: [[TestRunCommand]]
```

Comment:
What happens if there are newlines in this echo here? I suspect they might spill over and try to execute as commands.